web/default/style.css

```diff
@@ -819,10 +809,34 @@ a.scope            { /* scope              */ color: steelblue; font-weight: bol
     position: fixed;
     font-size: 12px;
     font-family: monospace;
-    overflow: auto;
+    overflow: hidden;
```

Comment:
if the content is clipped, what mechanism will add the scrollbar now ?

Reply:
The content is in the body and that is set to overflow auto